velox/docs/functions/spark/datetime.rst

```diff
@@ -22,6 +22,12 @@ These functions support TIMESTAMP and DATE input types.
 
     num_days can be positive or negative.
 
+.. spark:function:: dayofweek(date) -> integer
```

Comment:
should we also document that `dow` is an alias?

Reply:
Fixed